File: bakalarkaBE/bakalarkaBE/Models/VysetrenieZaznam.cs
using System;
using System.Collections.Generic;

namespace bakalarkaBE.Models;

public partial class VysetrenieZaznam
{
    public string Kod { get; set; } = null!;

    public int Idzaznam { get; set; }

    public DateOnly? Datum { get; set; }

    public virtual Zaznam IdzaznamNavigation { get; set; } = null!;

    public virtual Vysetrenie KodNavigation { get; set; } = null!;
}

File: FE/src/app/models/VysetrenieZaznam.ts
import {Zaznam} from "./Zaznam";
import {Vysetrenie} from "./Vysetrenie";

export class VysetrenieZaznam {
  kod: string = "";
  idzaznam: number = 0;
  datum:Date = new Date();
  vysledokvysetrenia: string = "";
  idzaznamNavigation!: Zaznam;
  kodNavigation!: Vysetrenie;
}
